src/modules/article.js

```diff
@@ -24,20 +24,72 @@ const createArticle = async (title, body) => {
     const response = await axios.post(
       "/admin/articles",
       {
-        article: 
-        {
+        article: {
           title: title,
           body: body
         }
       },
       {
         headers: headers
       }
-      );
+    );
     return response;
   } catch (error) {
-    return error.response
+    return error.response;
   }
 };
 
-export { getCurrentArticle, getArticles, createArticle };
+const getUnpublishedArticles = async () => {
+  let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
```

Comment:
the credentials are used in every request you added, can you extract this somehow so you don't need to have this one line in all of the methods?

Reply:
tried with axios.defaultheaders, did not work. I'll put this up on our chore list.